LUMO_toolbox/DOTHUB_LUMOdataQualityCheck.m

```diff
@@ -45,6 +46,11 @@ function DOTHUB_dataQualityCheck(nirsFileName,printFigFlag)
     printFigFlag = 1;
 end
 
+if ~exist('timeSelectFlag','var')
+    timeSelectFlag = 1;
```

Comment:
I would default to timeSelectFlag = 0.  i.e. if a user doesn't specify it, it is assumed we don't want to use it.

Reply:
Changed this